src/main/java/org/folio/circulation/domain/representations/CallNumberComponentsRepresentation.java

```diff
@@ -0,0 +1,28 @@
+package org.folio.circulation.domain.representations;
+
+import static org.folio.circulation.support.JsonPropertyWriter.write;
+
+import org.folio.circulation.domain.CallNumberComponents;
+
+import io.vertx.core.json.JsonObject;
+
+public class CallNumberComponentsRepresentation {
+
+  private CallNumberComponentsRepresentation() {
+    throw new UnsupportedOperationException("Do not instantiate");
```

Comment:
I personally omit these exceptions and just leave an empty private constructor. I accept folks could use reflection to invoke it.

Reply:
`AssertionError` is typically used for this. For example, `java.lang.Objects`:
 ```java
    private Objects() {
        throw new AssertionError("No java.util.Objects instances for you!");
    }
```
If you are adding this level of protection, you probably want to make the class `final` as well.

Reply:
Thanks @mreno-EBSCO, I've made the class final. I'd prefer  to stay with `UnsupportedOperationException`, because `AssertionError` looks for me a bit illogical. But the `UnsupportedOperationException` definitely says that instantiation is not supported for the class.

Reply:
Fine with me. Just don't tell Java's Josh Bloch, who wrote the book on this subject that you think `AssertionError` is illogical. 😉 

http://www.informit.com/articles/article.aspx?p=1216151&seqNum=4

Interestingly, I just looked at my first edition of his book (yeah, I bought it back when you had to buy tech books and, for some reason, I still have it) and he doesn't mention throwing `AssertionError` in the item (which was 3 in my edition, not 4), so he must have added this for the 2008 second edition. Most of the text is the same though. Hmm... also `AssertionError` didn't show up until 1.4 in 2002 (the first edition of _Effective Java_ was published in 2001). Well, I think I've beaten this to death...